### ISSUE TITLE
making the program accessible to blind people using a screen reader

### ISSUE THREAD

**Issue by paulbogo** [jwallet/spy-spotify#466]
Hello! would there be a way to label the checkboxes? I'm using a screen reader, I'm blind, and the whole advanced settings section is inaccessible to me, in the sense that I get the checkboxes, but without the label to figure out what each one means
Basically, all checkboxes in the program are unlabeled.
do you think this problem could be solved?

**Comment by jwallet** [comment]
So your screen reader is able to see "advanced settings" tab ? I use a old framework made by a dude to render the user interface, so I'm pretty sure he did not handle that. Windows form are kind of a thing of the pass, there are so better framework out there. Anyway, if you reader can read some labels, does it help to run the windows magnifier `Windows`&`+` key to zoom in, or `Windows`&`-` to zoom out

**Comment by jwallet** [comment]
[Debug.zip](https://github.com/jwallet/spy-spotify/files/10388360/Debug.zip)

I added accessibility info to all buttons, text inputs, and toggle (checkboxes), with an accessibility name, description and the type. So like I said it may not work if the guy that made the framework did not use what Microsoft provided for him when he built his WinForm framework. So if it does not work, and cannot do more than this sorry.

**Comment by paulbogo** [comment]
it's works. I will come back with a feedback. the last checkbox from advanced tab haven't label.

**Comment by jwallet** [comment]
feel free to submit new text and description for these options, i'm not sure how a screen reader reads it, I think it starts by saying the type of the element and its name, not even sure if it uses the description.

so if it reads everything it should say, `a "Push Button" named "Toggle mute ads" that "Will mute video and audio ads while recording"`

**Comment by paulbogo** [comment]
Thanks for work. most things work but, you could add rtbLog.TabStop = true;
 in frmEspionSpotify.cs. one checkbox it's left unlabeled, the one after toggle extra track title as id3 as subtitle...
thanks so much for your availability
You can add the labels how you did, it's works
